src/main/scala/org/broadinstitute/dsde/workbench/sam/util/JndiSupport.scala

```diff
@@ -27,6 +27,19 @@ trait JndiSupport {
     ctx.close()
     t.get
   }
+
```

Comment:
And that this is part of moving that other block of code from Access Policy to Directory?